client/modules/Run/__tests__/RunActions.spec.js

```diff
@@ -0,0 +1,102 @@
+import test from 'ava';
+import { actionTest } from 'redux-ava';
+import nock from 'nock';
+import configureStore from 'redux-mock-store';
+import thunkMiddleware from 'redux-thunk';
+import { API_URL } from '../../../util/apiCaller';
+
+import {
+  ADD_RUN,
+  DELETE_RUN,
+  ADD_RUNS,
+  addRun,
+  deleteRun,
+  addRuns,
+  fetchRun,
+  fetchRuns,
+  deleteRunRequest,
+} from '../RunActions';
+
+const run = { _id: 'id', results: ['result-set-1'], framework: 'Cypress', job: 'some-jenkins-job', runDate: '2019-01-28 16:10:56.346Z' };
+
+test('should return the correct type for addRun', actionTest(
+  addRun,
+  run,
+  { type: ADD_RUN, run },
+));
+
+test('should return the correct type for deleteRun', actionTest(
+  deleteRun,
+  run._id,
+  { type: DELETE_RUN, id: run._id },
+));
+
+test('should return the correct type for addRuns', actionTest(
```

Comment:
From my comment on RunActions, the export is necessary for these tests, however the tests below actually duplicate testing that the data returned from `addRuns` is formatted correctly (line 58 for example).  So if these were moved to private functions you'd still have the same coverage.